Makefile

```diff
@@ -40,7 +41,14 @@ _bootstrap-python-requirements:
 	# Ensure requirements.txt (used by Read the Docs) is in sync.
 	# Keep this separate from _bootstrap-python since the output changes based
 	# on the version of pipenv installed.
+	PIPENV_VERSION="$(shell ${PIPENV} --version | cut -d ' ' -f 3 | tr '.' ' ' | xargs printf '%02d')"; \
+	MIN_PIPENV_VERSION="$(shell echo ${MIN_PIPENV_VERSION} | tr '.' ' ' | xargs printf '%02d')"; \
+	if [[ $$PIPENV_VERSION -lt $$MIN_PIPENV_VERSION ]]; then \
+		echo "pipenv version ${MIN_PIPENV_VERSION} or later is required."; \
```

Comment:
Possible to add the command to update it here too?

Reply:
Done!